readme.md

```diff
@@ -1,104 +1,175 @@
 
-# Getting started with Metrics :beginner:
-
-We've tried to document all important decisions & approaches used for the development of this project. Reading each document is not an easy task and requires some time and patience. To help you get started, we collected the most useful documents that should help you to make fist steps:
-
-1. [Metrics developer configuration :gear:](docs/14_developer_configuration.md)
-2. [Why create a metrics platform? :thinking:](docs/01_design_doc.md)
-3. [GitHub Agile process :chart_with_upwards_trend:](docs/02_process.md)
-4. [Dart code style :nail_care:](docs/09_dart_code_style.md)
-5. [Collaboration :raised_hands:](docs/10_collaboration.md)
-
-Furthermore, every Metrics component requires component-specific documentation. Thus, to get started with the `CI integrations` it is recommended to get familiar with the following documents: 
-1. [CI integrations module architecture :building_construction:](metrics/ci_integrations/docs/01_ci_integration_module_architecture.md)
+<details>
+  <summary>Usage example</summary>
 
-Similarly, here is a list for the `Firebase` component:
-1. [Metrics firebase deployment :boat:](docs/08_firebase_deployment.md)
+Consider this short example on how to use the main `YamlMapParser` and `YamlMapFormatter` classes:
 
-In contrast with the above components, `Flutter Web` requires more steps to follow: 
-1. [Project metrics definitions :book:](docs/05_project_metrics.md)
-2. [Metrics Web Application architecture :walking:](metrics/web/docs/01_metrics_web_application_architecture.md)
-3. [Metrics Web presentation layer architecture :running:](metrics/web/docs/02_presentation_layer_architecture.md)
-4. [Widget structure organization :bicyclist:](metrics/web/docs/03_widget_structure_organization.md)
+```dart
+import 'package:yaml_map/src/yaml_map_formatter.dart';
+import 'package:yaml_map/src/yaml_map_parser.dart';
 
-# Guardian :shield:
+void main() {
+  const yaml = '''
+  foo:
+    bar:
+      baz: 1
+  ''';
 
-Detect flaky tests by analyzing JUnit XML files and orchestrate tools like Slack, Jira to notify the team.
+  const yamlMapParser = YamlMapParser();
+  final parsedYaml = yamlMapParser.parse(yaml);
 
-# Design :art:
+  print(parsedYaml); // {foo: {bar: {baz: 1}}}
+  print(parsedYaml['foo']); // {bar: {baz: 1}}
+  print(parsedYaml['foo']['bar']); // {baz: 1}
+  print(parsedYaml['foo']['bar']['baz']); // 1
 
-Follow the next links to get acquainted with the Metrics project design: 
-- [design/](design/)
+  final yamlFormatter = YamlMapFormatter();
+  print(yamlFormatter.format(parsedYaml));
+  // foo: 
+  //   bar: 
+  //     baz: 1
+}
+```
 
-# Documentation :books:
+</details>
 
-You can find complete Metrics documentation using the following links:
-- [general documentation](docs/)
-- [CI integrations documentation](metrics/ci_integrations/docs/)
-- [Web documentation](metrics/web/docs/)
+# :scroll: License
```

Comment:
Would it be useful to add a link for the dependencies licenses document here? What do you think?